public/mapfetch.js

```diff
@@ -0,0 +1,15 @@
+let myInput = document.querySelector('#myInput');
+let searchAbout = document.querySelector('#searchAbout');
+let mapLinkName = document.querySelector("#imgMap");
+
+
+
+searchAbout.addEventListener('click', function() {
+    // let url = `https://www.google.com/maps/search/?api=1&query=${myInput.value}+field`;
```

Comment:
clean up your code before submitting a PR: remove logs and comment code for example